src/module_library/AuxBioCro.cpp

```diff
@@ -732,16 +732,41 @@ soilML_str soilML(
  *  into the response of plant production to climate change?: development and
  *  experiments with WIMOVAC: (Windows Intuitive Model of Vegetation response
  *  to Atmosphere & Climate Change)" (University of Essex, 2002)
+ *
+ *  [YH] This function scales the assimilation rate. It should be called the growth
+ *  respiration instead of maintenance respiration, as defined in these papers:
+ *  - Apsim: (https://apsimdev.apsim.info/ApsimX/Documents/AgPastureScience.pdf)
+ *  - Thornley, J. H. M. "Growth, maintenance and respiration: a re-interpretation."
+ *    Annals of Botany 41.6 (1977): 1191-1203.
  */
-double resp(double base_rate, double mrc, double temp)
+double resp(double base_rate, double grc, double temp)
 {
-    double ans = base_rate * (1 - (mrc * pow(2, (temp / 10.0))));
+    double ans = base_rate * (1 - (grc * pow(2, (temp / 10.0))));
```

Comment:
Here's my rewrite using the `temperature_response_exponential` function. IDK if this is a good/correct use of `constexpr double variable = ...` the value of `log(2)/10` would only be needed at compile time here.

```c++
double resp(double base_rate, double grc, double temp){
  constexpr double coef = log(2)/10; 
  double gr = temperature_response_exponential(temp, 0, coef); 
  return base_rate *  ( 1- gr );
} 
```